docs/installation.md

```diff
@@ -77,7 +77,7 @@ Alternatively, you can install this from source by running:
 go install github.com/google/osv-scanner/cmd/osv-scanner@v1
 ```
 
-This requires Go 1.21.4+ to be installed.
+This requires Go 1.21.10+ to be installed.
```

Comment:
Should this be 11?

Reply:
I was following changes made from https://github.com/google/osv-scanner/pull/724/files 
But yeah, I'll update it to 11 for consistency